core/src/main/java/org/mule/context/DefaultMuleContextBuilder.java

```diff
@@ -92,6 +95,25 @@
 
     protected SplashScreen shutdownScreen;
 
+    public DefaultMuleContextBuilder()
+    {
+        this(false);
+    }
+
+    /**
+     * if {@code createSimpleRegistry} is {@code true}, then a {@link SimpleRegistry}
+     * will automatically be added to the created context. That only makes sense
+     * when trying to start a light weight context which doesn't require starting a heavier
+     * registry such as Spring or Guice or when testing. This is not something we would
+     * advice on production
+     *
+     * @param createSimpleRegistry whether or not to add a {@link SimpleRegistry} to the created context
+     */
+    public DefaultMuleContextBuilder(boolean createSimpleRegistry)
```

Comment:
Don't like this boolean here, feel it should be transparent.  MuleContext shoud have a SimpleRegisty/Limbo by default when it's created which gets replaces by SpringRegistry if it happens to be configured with Spring maybe?  Obisouly multiple registries made this easier, but that has it's own problems.